src/hooks/useCart/useCart.tsx

```diff
@@ -210,29 +210,26 @@ export const useCart = (
 
       let numUnverifiedTransactions = 0;
       let numIdentifiers = 0;
-      const identifierInputValues: string[] = [];
+      const identiferValues: string[] = [];
       const transactions = Object.values(cart)
         .filter(({ quantity }) => quantity)
-        .map(({ category, quantity, identifiers: identifierInputs }) => {
+        .map(({ category, quantity, identifiers }) => {
```

Comment:
Okay looks good to me! my initial idea was taking into consideration the possibility of doing a check for each category as well (i.e. Last voucher code > First voucher code), but not doing that liao haha

Reply:
I see!! Actually if we wanted to do per category, we can just keep a boolean, no need to keep both boolean and array too

Reply:
thats true!! hmm but array still needed to check uniqueness across categories right?

Reply:
Can use the boolean!! Very simplified example:

```
let isUnique = true
for item in cart:
  isUnique = isUnique && checkIsUnique(item.identifiers)
```